precise-revoke-redis-sessions/README.md

```diff
@@ -0,0 +1,30 @@
+# Precisely revoke sessions in Redis
+
+**Status: ✅ Ready to be used in production**
+
+## Description
+
+This script runs in `redis-cli`.
+It requires access to the Redis instances used by Grist.
+Because the session object in Redis is a JSON object, the script simplifies by just checking if the object contains the username.
+If the username is underspecified, it might delete sessions for other users with similar names.
```

Comment:
```suggestion
Because the session object in Redis is a JSON object, the script simplifies by just checking if the object contains the quoted username.
```

Reply:
LGTM!